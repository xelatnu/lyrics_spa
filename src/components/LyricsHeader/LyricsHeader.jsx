import SearchPanel from './components/LyricsSearchPanel/LyricsSearchPanel';

const LyricsHeader = ({ setLyricsData }) => {
  return (
    <header className="App-header">
      <h1>
        Search your favorit song lyrics
      </h1>
      <SearchPanel setLyricsData={setLyricsData} />
    </header>
  )
}

export default LyricsHeader;