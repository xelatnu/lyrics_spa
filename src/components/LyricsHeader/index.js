import LyricsHeader from "./LyricsHeader";

export default LyricsHeader;