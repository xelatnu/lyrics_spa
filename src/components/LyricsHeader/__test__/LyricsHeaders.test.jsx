/* eslint-disable react/jsx-filename-extension */
import { render } from '@testing-library/react';
import LyricsHeader from '../LyricsHeader';

describe('LyricsHeader', () => {
  it('should match a snapshot', () => {
    const { baseElement } = render(
      <LyricsHeader name="header">
        <p>Hello</p>
      </LyricsHeader>,
    );
    expect(baseElement).toMatchSnapshot();
  });
});
