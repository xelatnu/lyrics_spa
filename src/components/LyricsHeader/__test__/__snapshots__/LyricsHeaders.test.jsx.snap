// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LyricsHeader should match a snapshot 1`] = `
<body>
  <div>
    <header
      class="app-header"
    >
      <h1
        data-testid="header-1"
      />
      <h2
        title="Header"
      >
        Search your favorite song lyrics
      </h2>
      <div>
        <form>
          <input
            class="search-input "
            data-testid="search-input"
            name="lyrics"
            placeholder="Type lyrics"
            required=""
            type="text"
            value=""
          />
          <input
            class="search-input "
            name="artist"
            placeholder="Type artist"
            type="text"
            value=""
          />
          <button
            class="btn btn-primary"
            data-testid="submit-button"
            type="submit"
          >
            Search lyrics
          </button>
        </form>
      </div>
    </header>
  </div>
</body>
`;

exports[`LyricsHeader should match to snapshot 1`] = `
<body>
  <div>
    <header
      class="app-header"
    >
      <h1
        data-testid="header-1"
      />
      <h2
        title="Header"
      >
        Search your favorite song lyrics
      </h2>
      <div>
        <form>
          <input
            class="search-input "
            data-testid="search-input"
            name="lyrics"
            placeholder="Type lyrics"
            required=""
            type="text"
            value=""
          />
          <input
            class="search-input "
            name="artist"
            placeholder="Type artist"
            type="text"
            value=""
          />
          <button
            class="btn btn-primary"
            data-testid="submit-button"
            type="submit"
          >
            Search lyrics
          </button>
        </form>
      </div>
    </header>
  </div>
</body>
`;
